ui/marketplace/AppSecurityReport.tsx

```diff
@@ -44,23 +54,43 @@ const AppSecurityReport = ({ id, securityReport, showContractList, isLoading, on
     totalIssues = 0,
   } = securityReport?.overallInfo || {};
 
+  if ((!securityReport || !securityScore) && !isLoading) {
+    return null;
+  }
+
   return (
-    <Popover isOpen={ isOpen } onClose={ onClose } placement="bottom-start" isLazy>
+    <Popover isOpen={ isOpen } onClose={ onClose } placement={ popoverPlacement } isLazy>
       <PopoverTrigger>
         <SolidityscanReportButton
           score={ securityScore }
           isLoading={ isLoading }
           onClick={ handleButtonClick }
           isActive={ isOpen }
           onlyIcon={ onlyIcon }
-          label="The security score is based on analysis of a DApp's smart contracts."
+          label={ <>The security score is based on analysis<br/>of a DApp{ apos }s smart contracts.</> }
+          className={ className }
         />
       </PopoverTrigger>
       <PopoverContent w={{ base: '100vw', lg: '328px' }}>
         <PopoverBody px="26px" py="20px" fontSize="sm">
+          <Text fontWeight="500" fontSize="xs" mb={ 2 } variant="secondary">Smart contracts info</Text>
+          <Flex alignItems="center" justifyContent="space-between" h="32px">
```

Comment:
Is the hard-coded height of the element necessary here?

Reply:
This is the height specified in the design. I can change it to `py={ 1.5 }`, wdyt?